### DISCUSSION TITLE
Submarine pricing based on gun coverage, speed, and size.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5923] [Ideas and suggestions]
Rather than having sub builders set prices, I think there needs to be an option for automatic pricing based on the three factors mentioned in the title. A submarine with 360 degrees coverage, that is also fast and large should cost an arm and a leg. A sub that focuses on only one of the factors should be cheaper.

**Comment by ViktorShield** [comment]
What's the point? Sub makers are setting their own price manually anyway

**Reply by Cogs16** [reply]
Sub makers may not know what prices to set, and may set them too cheaply or make them too expensive.
